### PR TITLE
Made it possible to enable the Symfony2 404 handling with a config param...

### PR DESCRIPTION
**Pull request by rickprent**
....

### CODE REVIEW

File: test/Hostnet/HnDependencyInjectionPlugin/Symfony1FallbackTest.php [hunk 1]
```diff
@@ -64,10 +86,10 @@ public function testOnKernelExceptionSf2Init()
             ->expects($this->never())
             ->method('fallbackToSymfony1');
 
-        $event1 = $this->buildControllerEvent(array(clone $this->fallback, 'fallbackToSymfony1'));
+        $event1 = $this->buildControllerEvent([clone $this->fallback, 'fallbackToSymfony1']);
```
Comment (nicoschoenmaker) [inline test/Hostnet/HnDependencyInjectionPlugin/Symfony1FallbackTest.php]: For these square brackets, PHP5.4 is required.

You could remove 5.3 as a target in `.travis.yml`, and require PHP >=5.4 in `composer.json`, or revert back to `array()` notation.

**Comment by nicoschoenmaker** [comment]
@iltar We don't want to handle the Symfony 1 404 and 500 errors. We have an application where we've whitelisted every single route that should go to Symfony 2. So if there is a 404, we want it to be handled by Symfony 2.

**Comment by iltar** [comment]
@nicoschoenmaker @rickprent considering this is no bundle, I suppose the parameter check could work.

**Comment by nicoschoenmaker** [comment]
:+1:

**Pull request merged**